UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -772,58 +772,59 @@ private async void DataWindow_Closing(object sender, CancelEventArgs e)
                 e.Cancel = true;
 
                 bool save = false;
+                bool quit = false;
 
                 if (SettingsWindow.WarnOnClose)
                 {
-                    if (this.ShowQuestion("Are you sure you want to quit?") == MessageBoxResult.Yes)
-                    {
-                        if (this.ShowQuestion("Save changes first?") == MessageBoxResult.Yes)
-                        {
-                            if (scriptDialog is not null)
-                            {
-                                if (this.ShowQuestion("Script still runs. Save anyway?\nIt can corrupt the data file that you'll save.") == MessageBoxResult.Yes)
-                                    save = true;
-                            }
-                            else
-                                save = true;
+                    MessageBoxResult result = this.ShowQuestionWithCancel("Save changes before quitting?");
```

Comment:
removing the "are you sure you want to quit" is probably fine to do for now, but if you could add a comment that mentions to only ask whether changes should be saved if the file was changed, and ask whether the user really wants to quit otherwise would be nice.

Reply:
I don't know how to get if changes have been made; maybe that's for another PR.

Reply:
it absolutely is, was mostly just mentioning that adding a comment for that would be neat.